src/haddock/modules/analysis/ilrmsdmatrix/__init__.py

```diff
@@ -256,9 +294,12 @@ def _run(self):
         output_name = "ilrmsd.matrix"
         self._rearrange_output(
             output_name,
-            path=ilrmsd_obj.path,
+            path=Path("."),
             ncores=ncores
             )
+        # Delete the trajectory files
+        os.remove(rec_traj_filename)
+        os.remove(lig_traj_filename)
```

Comment:
This will raise an error if these files don't exist - either `try/except FileNotFoundError` or change `rec_traj_filename` and `lig_traj_filename` to `Path` then you could `if rec_traj_path.exists(): rec_traj_path.unlink()`